src/lib/hubspot/api.ts

```diff
@@ -69,15 +69,27 @@ export default class HubspotAPI {
 
   public async createEntities(kind: EntityKind, entities: NewEntity[]): Promise<ExistingEntity[]> {
     return await this.batchUpsert(kind, entities, async entities => {
-      const response = await this.apiFor(kind).batchApi.create({ inputs: entities });
-      return response.body.results;
+      try {
+        const response = await this.apiFor(kind).batchApi.create({ inputs: entities });
+        return response.body.results;
+      }
+      catch (e) {
+        this.console?.printError('HubSpot API', 'Error creating entities', { entities });
+        throw e;
+      }
     });
   }
 
   public async updateEntities(kind: EntityKind, entities: ExistingEntity[]): Promise<ExistingEntity[]> {
     return await this.batchUpsert(kind, entities, async entities => {
-      const response = await this.apiFor(kind).batchApi.update({ inputs: entities });
-      return response.body.results;
+      try {
+        const response = await this.apiFor(kind).batchApi.update({ inputs: entities });
+        return response.body.results;
+      }
+      catch (e) {
+        this.console?.printError('HubSpot API', 'Error updating entities', { entities });
```

Comment:
I think we also need to log `kind` here and in other spots don't we?

Reply:
This logs the properties that will be upserted which will probably be enough. But I'll add the kind anyway just in case.